src/main/java/XbeLoader/XbeXbSymbolDatabaseAnalyzer.java

```diff
@@ -92,14 +92,20 @@ public boolean added(Program program, AddressSetView set, TaskMonitor monitor, M
 			return false;
 		}
 		String xbePath = program.getExecutablePath();
+		// HACK: Somehow GUI broke this yet headlessAnalyzer didn't...
+		// Even if remove first forward slash before drive letter cause corruption to "Executable Location" meta, not even sure why either.
+		// Plus extra check if using network drive for safety precaution.
+		if (Platform.CURRENT_PLATFORM.getOperatingSystem() == OperatingSystem.WINDOWS && xbePath.toCharArray()[0] == '/' && xbePath.toCharArray()[1] != '/') {
```

Comment:
nits:
* Could you please shorten/wrap this line please
* Might change `xbePath.toCharArray()[0] == '/'` to `(xbePath.charAt(0) == '/')`

Reply:
Done, it's been a long time I had touched java coding. Yet still not an expert with it.